examples/MODULE.bazel

```diff
@@ -4,21 +4,98 @@ module(
     compatibility_level = 1,
 )
 
+bazel_dep(name = "platforms", version = "0.0.11")
+
 bazel_dep(name = "rules_cuda", version = "0.2.3")
 local_path_override(
     module_name = "rules_cuda",
     path = "..",
 )
 
 cuda = use_extension("@rules_cuda//cuda:extensions.bzl", "toolchain")
+cuda.redist_json(
```

Comment:
I extended the examples to test this. We will probably need something in `tests/integration`?